01-messaging.md

```diff
@@ -331,24 +335,27 @@ The fundee node:
     - MUST use `temporary_channel_id` in lieu of `channel_id`.
 
 A sending node:
-  - when sending `error`:
-    - MUST fail the channel referred to by the error message.
   - SHOULD send `error` for protocol violations or internal errors that make channels unusable or that make further communication unusable.
   - SHOULD send `error` with the unknown `channel_id` in reply to messages of type `32`-`255` related to unknown channels.
+  - when sending `error`:
+    - MUST fail the channel(s) referred to by the error message.
+  - when sending `warning`:
+	- MAY set `channel_id` to all zero if the warning is not related to a specific channel.
+  - MAY close the connection after sending.
```

Comment:
Why would that not hold for `error`? I understood `error` to be the case that that a force close should trigger. I see good reasons that after publishing the commitment transaction a peer might want to close the connection.